README.md

```diff
@@ -15,6 +13,19 @@
 
 Please see the individual package READMEs for details on how to install and use them.
 
+## Running the static Webviz application
+
+We recommend using the [hosted version of Webviz](https://webviz.io/app/?demo), which can connect to your [rosbridge_server](http://wiki.ros.org/rosbridge_suite/Tutorials/RunningRosbridge) or stream in bag data from your S3/GCS bucket. This way you'll always use the latest version of Webviz.
+
+However, sometimes the hosted version is inconvenient, when streaming data from robots on the field where there is poor internet connectivity. For this it is useful to use a static build of webviz.
+
+```sh
+npm run bootstrap # install dependencies
+npm run build # build all packages
+npm run build-static-webviz # generate static build in __static_webviz__
+npm run serve-static-webviz # serve static build on localhost:8080
```

Comment:
could we just reduce this to one command that chains them all together? Especially since so many folks are coming from a non-nodejs background, I feel like it would be a lot less distracting to make this a one-command app.

Reply:
Yeah, I think in effect that's what the Docker image is going to be. Once I figure out automated builds, I'll add a section here that just has one command for running the Docker image, and then these `npm` commands are the more manual version if you want to get in the weeds of it yourself (in which case it's useful to know what the different underlying commands do).